editor/script_editor_debugger.cpp

```diff
@@ -493,23 +493,31 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 			pinfo.usage = PropertyUsageFlags(int(prop[4]));
 			Variant var = prop[5];
 
+			Variant::Type TYPE = var.get_type();
```

Comment:
Seems unused.

Reply:
Yup, that was a remnant from debugging. Removed it.